packages/mgt-chat/src/statefulClient/StatefulGraphChatListClient.ts

```diff
@@ -399,19 +405,19 @@ detail: ${JSON.stringify(eventDetail)}`);
   /*
    * Event handler to be called when a new message is received by the notification service
    */
-  private readonly onMessageReceived = async (message: ChatMessage) => {
+  private readonly onMessageReceived = (message: ChatMessage) => {
     if (message.chatId) {
       this.notifyChatMessageEventChange({ message, type: this.getSystemMessageType(message) });
 
       // await this._cache.cacheMessage(message.chatId, message);
-      const messageConversion = this.convertChatMessage(message);
-      const acsMessage = messageConversion.currentValue;
-      this.updateMessages(acsMessage);
-      if (messageConversion.futureValue) {
-        // if we have a future value then we need to wait for it to resolve before we can send the read receipt
-        const futureMessageState = await messageConversion.futureValue;
-        this.updateMessages(futureMessageState);
-      }
+      // const messageConversion = this.convertChatMessage(message);
```

Comment:
did you want this commented out? or can it be deleted?

Reply:
There are some manipulation code here around messages, I am not sure if would be useful to keep a reference here for now? I can remove it if we think otherwise.

Reply:
If you think it is helpful, leave it and we can remove later.